README.md

```diff
@@ -277,7 +342,7 @@ As already discussed, employees can get restricted stock, stock options, or RSUs
 - If you receive an RSU when the stock is of little value, you cannot elect to be taxed on the value of that stock when you receive the RSU—you pay taxes at vesting time, based on the value of the shares at that time.
 - 🔸 There is a combination of big problems for RSUs in private companies:
     - You will owe tax when you receive the shares, even though they are illiquid.
-    - You can't minimize the impact of an increase in value of the underlying shares between the date you receive the RSU and the date it is settled.
+    - You can’t minimize the impact of an increase in value of the underlying shares between the date you receive the RSU and the date it is settled.
```

Comment:
Do you mean the tax impact? Is that what an 83b does? I think it's helpful to clarify and reinforce.